.netlify/build/chunks/2024-a-year-in-review_BVxUPuD3.mjs

```
@@ -0,0 +1,314 @@
+  }) : _createMdxContent(props);
+}
+
+const url = "src/content/blog/2024/12/31/2024-a-year-in-review.mdx";
+const file = "/home/rosnovsky/code/rosnovsky.us/src/content/blog/2024/12/31/2024-a-year-in-review.mdx";
+const Content = (props = {}) => MDXContent({
+  ...props,
+  components: { Fragment: Fragment, ...props.components, "astro-image":  props.components?.img ?? $$Image },
+});
+Content[Symbol.for('mdx-component')] = true;
+Content[Symbol.for('astro.needsHeadRendering')] = !Boolean(frontmatter.layout);
+Content.moduleId = "/home/rosnovsky/code/rosnovsky.us/src/content/blog/2024/12/31/2024-a-year-in-review.mdx";
```

Biome (1.9.4)

[error] Avoid redundant `Boolean` call

It is not necessary to use `Boolean` call when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant `Boolean` call


(lint/complexity/noExtraBooleanCast)